src/main/java/br/com/swconsultoria/nfe/util/WebServiceUtil.java

```diff
@@ -108,6 +105,9 @@ public static String getUrl(ConfiguracoesNfe config, DocumentoEnum tipoDocumento
                     secao = tipoDocumento.getTipo() + "_SVC-AN_"
                             + (config.getAmbiente().equals(AmbienteEnum.HOMOLOGACAO) ? "H" : "P");
                 }
+            }else if (!tipoServico.equals(ServicosEnum.URL_CONSULTANFCE)
```

Comment:
Não vai entrar aqui quando for contingência, mas não poderia entrar no if acima se for NFC-e.

Não existe contingência SVC para NFC-e, deve ser por isso que estava invertido.

`ServicosEnum.URL_CONSULTANFCE` e `ServicosEnum.URL_QRCODE` tem que olhar a "Sefaz Original" e não a SVC.

para testar o problema é só usar `config.setContigenciaSVC(true)` e tentar emitir uma NFC-e.

Há dois caminhos, a lib entender que SVC + NFC-e não faz sentido (ao não tentar SVC quando for `URL_CONSULTANFCE` / `URL_QRCODE` ) OU quem consome a Lib controlar para que não defina `ContigenciaSVC` quando for NFC-e.

Não sei o que o @Samuel-Oliveira pensa sobre isso.

Reply:
@Samuel-Oliveira faz sentido pra você?

Ao invés disso:  
```java   
} else if (config.isContigenciaSVC()) {
     ...
} else if (!tipoServico.equals(ServicosEnum.URL_CONSULTANFCE)
             && !tipoServico.equals(ServicosEnum.URL_QRCODE) && ObjetoUtil.verifica(url).isPresent()) {
     ...
}     
```
  
Usar isso:
```java
} else if (!tipoServico.equals(ServicosEnum.URL_CONSULTANFCE)
             && !tipoServico.equals(ServicosEnum.URL_QRCODE) 
             && config.isContigenciaSVC()) {
     ...
} else if (!tipoServico.equals(ServicosEnum.URL_CONSULTANFCE)
             && !tipoServico.equals(ServicosEnum.URL_QRCODE) && ObjetoUtil.verifica(url).isPresent()) {
     ...
}     
```

Reply:
Olá @jardelnovaes, faz sentido sim. 
Eu ainda incluiria uma verificação para garantir que somente NF-e entre em contingência.

Reply:
Olá @jardelnovaes, faz sentido sim.
Eu ainda incluiria uma verificação para garantir que somente NF-e busque URLs em contingência.
Ficaria assim: 
```java

} else if (!tipoServico.equals(ServicosEnum.URL_CONSULTANFCE)
             && !tipoServico.equals(ServicosEnum.URL_QRCODE) 
             && config.isContigenciaSVC() && tipoDocumento.equals(DocumentoEnum.NFE)) {
     ...
} else if (!tipoServico.equals(ServicosEnum.URL_CONSULTANFCE)
             && !tipoServico.equals(ServicosEnum.URL_QRCODE) && ObjetoUtil.verifica(url).isPresent()) {
     ...
}     

``` 

Vamos aguardar para saber o que o @Samuel-Oliveira acha.